app/views/notes/_nav_links.html.erb

```diff
@@ -0,0 +1,34 @@
+   <ul class="nav nav-pills notes-pills">
+     <li class="nav-item">
+       <a class="nav-link <% if params[:controller] == "notes" %> active <% end %>" href="/notes/">
```

Comment:
Could it be that we need to add another condition for this "active" state here?

Reply:
Thank you @jywarren  and @RuthNjeri.
It turned out to be a syntax error in the test :P 
It should be working now. Also, I made the test simpler this time.

Reply:
awesome, good catch!